### DISCUSSION TITLE
Fading between songs

### DISCUSSION THREAD

**Discussion by DaniDipp** [jagrosh/MusicBot#642] [Ideas]
My community and I would love it if the bot fade between songs in the queue instead of doing a hard cut. This would bring an extra layer of immersion to using the bot for background music :)

**Comment by DemonFiend** [comment]
i agree that this would be a pretty cool feature, Even add a configurable change to allow something like 0-5s for the fade so users who want the hard cut could still have it & allow DJ's to change it live. +1
